next.config.js

```diff
@@ -39,8 +39,8 @@ const nextConfig = {
   },
   rewrites: async () => [
     {
-      source: '/api/segment',
-      destination: 'https://api.segment.io/v1/batch',
+      source: '/api/analytics/:path',
+      destination: 'https://nearpavelsqp.dataplane.rudderstack.com/:path*',
```

Comment:
Rudderstack appends extra paths to specify what it is tracking e.g `https://nearpavelsqp.dataplane.rudderstack.com`+`/v1/page` tracks page events.

So we need to forward paths to the destination